### DISCUSSION TITLE
Support for Multiple Topic Operator in the same Namespace.

### DISCUSSION THREAD

**Discussion by somasundaram1996** [strimzi/strimzi-kafka-operator#8065] [Q&A]
Hi Team,

I have a scenario where I have to setup two Kafka Cluster in the same namespace each one will serve a different use case. But when I went through the Documentation it is not possible to have two Kafka cluster in the same namespace with Topic Operator Enabled which will cause the issue in internal topics and some inconsistency problem. 

Is there any roadmap that is planned for this specific use case.

I have a suggestion, Where we can include the cluster-name in the resource name and make the topicName in the spec to the name that was intented.

For Eg:
**Topic Name:** my-topic
**Cluster to Create Topic:** my-cluster
**Kubernetes KafkaTopic resource Name:**  my-cluster-my-topic
`
apiVersion: kafka.strimzi.io/v1beta2
kind: kafkaTopic
metadata:
  name: my-cluster-my-topic
  labels:
    strimzi.io/cluster: my-cluster
spec:
  topicName: my-topic
  # ...
`

**Comment by scholzj** [comment]
I think the key is to keep things simple by using different namespaces. This just overcomplicates things for everyone else who uses different namespaces, cause backward compatibility issues with existing topic names, possibly causes issues with the length of the name etc. So I think just sticking to different namespaces is much easier.
